scripts/template_config.yaml

```diff
@@ -0,0 +1,47 @@
+# This document is intended to serve as a template for constructing new
+# configuration YAMLs for use with the `simulation.py` script.
+
+# XXX do we want this to be an exhaustive template (all possible things
+# specified), or do we want this to be a minimal template?
+
+bda:
+    apply_bda: True
+    max_decorr: 0
+    pre_fs_int_time: !dimensionful 
+        value: 0
+        units: 's'
+    corr_FoV_angle: !dimensionful
+        value: 0
+        units: 'deg'
+    max_time: !dimensionful
+        value: 0
+        units: 's'
+    corr_int_time: !dimensionful
+        value: !!null
+        units: 's'
+defaults:
+    apply_defaults: False
```

Comment:
does it make sense to specify defaults and not apply them?

Reply:
not at all; will update before next push.